tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,1251 @@
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+
+        for rule_name, expected_content in expected_rule_contents.items():
+            expect_acl_rule_match(duthost, rule_name, expected_content)
+
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+
+def dynamic_acl_apply_drop_scale_rules(duthost, setup):
+    """Apply a large amount of drop rules to the duthost"""
+
+    priority = MAX_DROP_RULE_PRIORITY
+    json_patch = []
+    expected_rule_contents = {}
+    rule_number = 1
+
+    for port_name in setup["scale_port_names"]:
+        rule_name = "DROP_RULE_" + str(rule_number)
+        full_rule_name = "/ACL_RULE/DYNAMIC_ACL_TABLE|"+rule_name
+        rule_vals = {
+            "PRIORITY": str(priority),
+            "PACKET_ACTION": "DROP",
+            "IN_PORTS": port_name
+        }
+        patch = {
+            "op": "add",
+            "path": full_rule_name,
+            "value": rule_vals
+        }
+        json_patch.append(patch)
+        expected_content = ["DYNAMIC_ACL_TABLE", rule_name, str(priority), "DROP", "IN_PORTS: " + port_name, "Active"]
+        expected_rule_contents[rule_name] = expected_content
+        priority -= 1
+        rule_number += 1
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+
+        for rule_name, expected_content in expected_rule_contents.items():
+            expect_acl_rule_match(duthost, rule_name, expected_content)
+
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+
+def dynamic_acl_remove_ip_forward_rule(duthost, ip_type):
+    """Remove selected forward rule from the acl table"""
+
+    if ip_type == "IPV4":
+        rule_name = "RULE_1"
+    else:
+        rule_name = "RULE_2"
+
+    extra_vars = {
+        "rule_name": rule_name
+    }
+
+    output = format_and_apply_template(duthost, REMOVE_RULE_TEMPLATE, extra_vars)
+
+    expect_op_success(duthost, output)
+
+    expect_acl_rule_removed(duthost, rule_name)
+
+
+def dynamic_acl_remove_table(duthost):
+    """Remove an ACL Table Type from the duthost"""
+
+    output = load_and_apply_json_patch(duthost, REMOVE_TABLE_FILE)
+
+    expect_op_success(duthost, output)
+
+
+def dynamic_acl_remove_nonexistent_table(duthost):
+    """Remove a nonexistent ACL Table from the duthost, verify it fails"""
+
+    output = load_and_apply_json_patch(duthost, REMOVE_NONEXISTENT_TABLE_FILE)
+
+    expect_op_failure(output)
+
+
+def dynamic_acl_remove_table_type(duthost):
+    """Remove an ACL Table definition from the duthost"""
+
+    output = load_and_apply_json_patch(duthost, REMOVE_TABLE_TYPE_FILE)
+
+    expect_op_success(duthost, output)
+
+
+def test_gcu_acl_arp_rule_creation(rand_selected_dut,
+                                   ptfadapter,
```

Comment:
In order to be compatible for dualtor setup, please call fixture `toggle_all_simulator_ports_to_rand_selected_tor` for each test case. This is to ensure the DUT we run the test are active.

Reply:
Can take https://github.com/sonic-net/sonic-mgmt/blob/master/tests/everflow/test_everflow_testbed.py as a reference.